### PR TITLE
[Bug] Create dotenv file

### PR DESCRIPTION
**Pull request by Onyimatics**
### What does this PR do?
- This PR creates a dotenv file and adds the environment variables to it

#### Type of change
- [ ] Feature
- [x] Bug
- [ ] Chore

#### Description of Task to be completed?
- Create dotenv file
- Create dotenv.example file
- Hide firebase credentials

#### Checklist:
- [x] Create dotenv file
- [x] Create dotenv.example file
- [x] Hide firebase credentials

#### Stories?
- [Link to story](https://trello.com/c/sBFphNer/54-convert-the-firebase-config-file-into-environmental-variables)

### CODE REVIEW

**Review by xand6r — APPROVED** [review]
LGTM

**Review by usmanoa — APPROVED** [review]
Looks good.

File: src/components/login/components/Login.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ import {
 } from 'antd';
 import { useDispatch, useSelector } from 'react-redux';
 
-import firebase from 'firebase';
+import firebase from 'firebase/app';
+import 'firebase/auth';
 import { Redirect } from 'react-router-dom';
 import {
     SIGNINWITHACCOUNT, SIGNIN, CONFIRM_HEADER, CONFIRM, LOADING_MESSAGE
```
Comment (EngrEric) [inline src/components/login/components/Login.js:14]: separate imports from relative paths and the imports from npm modules. something like
```
import { useDispatch, useSelector } from 'react-redux';
import { Redirect } from 'react-router-dom';
import firebase from 'firebase/app';
import 'firebase/auth';

import {
    SIGNINWITHACCOUNT, SIGNIN, CONFIRM_HEADER, CONFIRM, LOADING_MESSAGE ...
```

**Review by EngrEric — APPROVED** [review]
LGTM

File: src/components/login/components/Login.js (hunk 1 above)
Comment (Onyimatics) [reply src/components/login/components/Login.js:14]: Alright @EngrEric . I will do that. Thanks

**Comment by xand6r** [comment]
[Convert the firebase config file into environmental variables](https://trello.com/c/sBFphNer/54-convert-the-firebase-config-file-into-environmental-variables)

**Comment by xand6r** [comment]
[Convert the firebase config file into environmental variables](https://trello.com/c/sBFphNer/54-convert-the-firebase-config-file-into-environmental-variables)

**Pull request merged**